worker-core/src/main/java/com/hpe/caf/worker/core/WorkerCore.java

```diff
@@ -252,6 +252,18 @@ private void validateTaskMessage(TaskMessage tm) throws InvalidTaskException
 
         private boolean isTaskIntendedForThisWorker(final TaskMessage tm, final TaskInformation taskInformation)
         {
+            if (!Boolean.parseBoolean(
+                    System.getProperty("CAF_WORKER_ENABLE_DIVERTED_TASK_CHECKING", "true"))) {
```

Comment:
Do a `public static final` so that this isn't re-parsed for every message.

Like this one: [WorkerTaskImpl.java:49-50](https://github.com/WorkerFramework/worker-framework/blob/v5.1.0/worker-core/src/main/java/com/hpe/caf/worker/core/WorkerTaskImpl.java#L49-L50)